doc/tutorials/run_graph_states.py

```diff
@@ -205,3 +207,31 @@
 
 viz.plot_params["figure.figsize"] = (5.4, 4)
 viz.plot_mat_heat_map(adj)
+
+######################################################################
+# Now that we know how to create custom cluster states in ``flamingpy``,
+# let's use some built-in functions to create some well known graph states.
+# First, we need to import the module ``graph_states`` from ``flamingpy.utils``:
+
+from flamingpy.utils import graph_states
+
+viz.plot_params["figure.figsize"] = (8, 6)
+
+######################################################################
+# Some of the state families that we have access to are star states, GHZ states,
+# linear states, and ring states. Let begin by plotting the star state 
+# of 10 qubits (this state is LU-equivalent to a GHZ state of 10 qubits):
+
+
+graph_states.star(10).draw(**drawing_opts)
```

Comment:
```suggestion
graph_states.star_graph(10).draw(**drawing_opts)
```
(To align with suggested change below)